Tests/SourceKitDTests/CrashRecoveryTests.swift

```diff
@@ -312,4 +312,40 @@ final class CrashRecoveryTests: XCTestCase {
       "Clangd restarted too quickly after crashing twice in a row. We are not preventing crash loops."
     )
   }
+
+  func testPullDiagnosticsAfterCrash() async throws {
+    try SkipUnless.longTestsEnabled()
+
+    // Ensure that we don't cash the diagnostic report from the crashed sourcekitd and don't emit any of the internal
```

Comment:
```suggestion
    // Ensure that we don't cache the diagnostic report from the crashed sourcekitd and don't emit any of the internal
```

But I wouldn't restart testing for it 😅